src/data_pipeline_api/simple_network_sim_api.py

```diff
@@ -0,0 +1,13 @@
+from io import TextIOWrapper
+from pathlib import Path
+from pandas import read_csv, DataFrame
+from data_pipeline_api.file_api import FileAPI
+
+class SimpleNetworkSimAPI(FileAPI):
```

Comment:
Yes I greatly prefer composition, but I left it like this precisely so that people could access those methods in this uncertain state - once we have pinned things down I think we should do what you suggest.